src/nyx/features/contour.cpp

```diff
@@ -43,9 +51,34 @@ void ContourFeature::buildRegularContour(LR& r)
 	{
 		auto x = px.x - minx + 1, 
 			y = px.y - miny + 1;
-		paddedImage [x + y * (width + 2)] = px.inten;
+		paddedImage [x + y * (width + 2)] = 1;	// Building the contour around the whole ROI mask image
 	}
 
+	#if 0
+	//

```

Comment:
is it possible to remove this dead code?

Reply:
I intentionally left disabled diagnostic code that will help me unwind a weird result for a user (and had helped many times) instead of keeping a bunch of diagnostic snippets in my own computer. Ok, removing the diagnostic code : (

Reply:
Ok, I decided to keep the diag code, as level-4 verbose output